addons/l10n_it_edi/wizard/account_move_send.py

```diff
@@ -1,12 +1,177 @@
 # -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models
+from base64 import b64encode
+from markupsafe import escape
+
+from odoo import _, api, fields, models, SUPERUSER_ID
+from odoo.exceptions import UserError
+from odoo.addons.base.models.ir_qweb_fields import nl2br
+from odoo.addons.l10n_it_edi.tools.xml_utils import format_errors
+from odoo.addons.account_edi_proxy_client.models.account_edi_proxy_user import AccountEdiProxyError
 
 
 class AccountMoveSend(models.Model):
     _inherit = 'account.move.send'
 
-    def _get_mail_attachment_from_doc(self, doc):
-        if doc.edi_format_id.code == 'fattura_pa':
-            return []
-        return super()._get_mail_attachment_from_doc(doc)
+    enable_l10n_it_edi = fields.Boolean(compute='_compute_fields_from_moves_state')
```

Comment:
I suppose there is a reason for the reversal: enable_l10n_it_edi instead of l10n_it_edi_enable

Reply:
Sure, consistency with:
```
enable_send_mail
checkbox_send_mail
enable_download
checkbox_download
enable_send_by_post
checkbox_send_by_post
```
Ask Laurent about that.

Reply:
... when someone does a code convention I don't agree with, then I comply to it, and I get the complaints anyway hehe 😋

Reply:
@lordkrandel Hm, would not the convention be l10n_it_enable_fatturapa?

Reply:
@smetl

Reply:
Would you please stop using "FatturaPA" as a term... it really is the name of the XML format only, and it's a stupid name, it's like "Public Administration Invoice" - it's ... completely wrong, since we also do tax integrations on vendor bills that are B2B and... 

Just forget "FatturaPA" as a term.

Reply:
`l10n_it_edi_enable_send` would be consistent with mexico and OK then, I think.